src/main/resources/api/service_openapi.yaml

```diff
@@ -137,6 +134,26 @@ paths:
             schema:
               type: string
                     
+      responses:
+        200:
+          $ref: '#/components/responses/QueryCreated'
+
+  /api/v1/query/{id}:
```

Comment:
Another approach here would be to put all the functionality under `/query` (or maybe `queries`). The existing query APIs could be rewritten as `POST` operations on `/query`, as those create a query. Under that approach, the current APIs could be left as-is (but rewritten internally to use the new code) for backward compatibility.